src/apify/scrapy/cache.py

```diff
@@ -0,0 +1,176 @@
+from __future__ import annotations
```

Comment:
Taking account the current project structure and the fact that this is an [Extension](https://docs.scrapy.org/en/latest/topics/extensions.html) (correct me if I am wrong), add it please to the `extensions/` subpackage with `__init__.py` exposing it.

Reply:
`extensions/_httpcache.py` and `extensions/__init__.py` exposing `ApifyCacheStorage` please

Reply:
So the recommended way to import the class is `apify.scrapy.extensions.httpcache.ApifyStorage` or `apify.scrapy.extensions.ApifyStorage`? So that I know how to adjust docs and the module path in the apply settings function.